build/charts/theia/provisioning/datasources/migrators/000005_0-6-0.down.sql

```diff
@@ -212,3 +212,9 @@ ALTER TABLE tadetector_local
     DROP COLUMN aggType,
     DROP COLUMN direction,
     DROP COLUMN podName;
+ALTER TABLE flows
```

Comment:
Should these changes be in this file? Or it should be 000006_0-7-0?

Reply:
Changed